icons/tools/build-visio-stencil-creator.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+GITHUB_ORG="hoveytechllc"
+REPO_NAME="visio-stencil-creator"
+TAG_NAME="v1.0"
+
+GCR_REPO="gcr.k8s.io/${REPO_NAME}:${TAG_NAME}"
```

Comment:
This has to be `staging-k8s.gcr.io/...` so I can push it.  promoter moves it to final location.